src/ipv4.rs

```diff
@@ -309,6 +309,12 @@ mod test {
     }
 
     #[test]
+    fn parse_v4_fail_2slashes() {
```

Comment:
Would you mind changing this to `parse_v4_fail_two_slashes`, please?

Reply:
done